README.md

```diff
@@ -10,24 +10,39 @@
 
 The Service Provider Dashboard is a dashboard application where
 [SURFconext](https://www.surf.nl/diensten-en-producten/surfconext/index.html) Service Providers can register and manage
-their services. This can be both SAML 2.0 and OpenID Connect entities.
+their services. This can be both SAML 2.0, OpenID Connect Relying Parties and Oauth 2.0 Resource Server entities.
 
 ## Prerequisites
 
-- [PHP](https://secure.php.net/manual/en/install.php) (5.6 or higher)
+- [PHP](https://secure.php.net/manual/en/install.php) (7.2)
 - [Composer](https://getcomposer.org/doc/00-intro.md)
 - [Apache Ant](https://ant.apache.org/manual/install.html)
 - [Ansible](https://docs.ansible.com/ansible/intro_installation.html)
 - [Vagrant](https://www.vagrantup.com/docs/installation/)
   - Optional, but recommended: [Hostsupdater plugin](https://github.com/cogitatio/vagrant-hostsupdater)
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 
+As of release 2.8 the Vagrant + Ansible dev environment has been discontinued in favour of a docker-compose installable
+dev machine. Instructions below should still help you in building a Vagrant based dev env, but this will be removed from
+the project in the next release. For now use `docker-compose up -d` to create and build the development environment.
+
+An entry in your hostsfile is still required for things to work. An example entry would look like:
+
+```
+127.0.0.1 welcome.vm.openconext.org static.vm.openconext.org mujina-sp.vm.openconext.org mujina-idp.vm.openconext.org engine-api.vm.openconext.org oidc.vm.openconext.org manage.vm.openconext.org spdashboard.vm.openconext.org
```

Comment:
@quartje which apps actually run on the base openconext docker image? I stripped some that I knew where not on there, but can you further validate this hostsfile entry?

Reply:
I'll open another PR to document the Docker changes.